### PR TITLE
Added dwv with pan, zoom, and slice functionality, fixed relative pat…

### PR DESCRIPTION
**Pull request by kenannajjar**
…h issue

### CODE REVIEW

**Comment by avaidyam** [comment]
Looks good! I see some weird behavior with cursor selection: as you zoom/pan/scroll the cursor inadvertently causes a text highlight around the whole imaging viewer. I’d also see if it’s possible to make the toolbar with the tools appear at the cursor on image right click (like the IBM viewer).
---
Ref #28

**Comment by kenannajjar** [comment]
> ## Looks good! I see some weird behavior with cursor selection: as you zoom/pan/scroll the cursor inadvertently causes a text highlight around the whole imaging viewer. I’d also see if it’s possible to make the toolbar with the tools appear at the cursor on image right click (like the IBM viewer).
> 
> Ref #28

Just pushed changes that should fix this issue

File: src/components/patient/tabs/chartreview/ImagingTabContent.js [hunk 1]
```diff
@@ -29,188 +32,9 @@ let loadImages = (scanID) => {
...
-
-    // Clean up our event listeners when removing our RadiologyCanvas:
-    return(() => {
-      canvas.removeEventListener('wheel', handleScroll);
-      canvas.removeEventListener('pointermove', handleMouseMotion);
-      canvas.removeEventListener('pointerdown', handleMouseDown);
-      ctxRef.current.clearRect(0, 0, canvas.width, canvas.height);
-    });
-  }, [props.mode]);
+// The old RadiologyCanvas and ImageViewer components can be kept for reference but will not be used if DWV is fully integrated.
```
Comment (avaidyam) [inline src/components/patient/tabs/chartreview/ImagingTabContent.js:35]: I say let's delete the old code now

**Comment by avaidyam** [comment]
[Can you update the code here](https://github.com/avaidyam/ProjectEHR/blob/f6e3dc9652b8b847560910831131b005c65db757/src/components/patient/tabs/chartreview/ChartReviewTabContent.js#L92) so that the imaging tab contents load the JPEG in the new DICOM viewer? The specialty tests tab can continue to load an unrelated random sample file for now.

**Pull request merged**